include/mbedtls/config.h

```diff
@@ -1779,6 +1779,13 @@
  */
```

Comment:
Commit "Add I/O buffer length fields to mbedtls_ssl_context" needs Darryl's signed-off-by as he was the primary author. You should have your signed-off-by as well if you changes the code, for a total of two signed-off-by lines. Same goes for the other commit from Darryl.

Reply:
Should I sign it off for @dgreen-arm? Or should he prepare such commit?